app/base/handlers.js

```diff
@@ -7,7 +7,7 @@ const uuid = require('uuid');
 
 
 function sso_logout_url() {
-  const returnTo = encodeURI(`${config.app.protocol}://${config.app.host}:${config.app.port}`);
+  const returnTo = encodeURI(`${config.app.protocol}://${config.app.host}`);
```

Comment:
Should this be removed completely? I think maybe it should be possible to set it to `null` or `undefined` and only add it to the `returnTo` URL if it is set to a number.

Reply:
I think omitting it would break logout when running locally

Reply:
As mentioned in PR description, when running the UI locally, just add it to the `APP_HOST`, in fact the default value is `localhost:3000`.